### ISSUE TITLE
Loop on windows login when deployed on IIS

### ISSUE THREAD

**Issue by LoicVandersmissen** [d00lar/BlazorWasmAuthStateProvicerWindowsAuthAndJWT#1]
Hello, 
your approch is pretty good for my needs but  when my customers deployed solution on IIS the windows login popup appear in loop.
Did you try IIS deployement? 
If yes, does it need specific parameter on IIS site?
Best regards

**Comment by d00lar** [comment]
hi 
it was long time ago but it worhed in iis also 4shure
both anonymous and windows auth in iss must be enabled

**Comment by krunalshah10287** [comment]
This works fine. But one issue I found in this implementation is when JWT token expired and when we try to call the api. it's not working. it returns 401 unauthorized error. Is there any specific setting I need to do to update the token once it's expired.

**Comment by d00lar** [comment]
yes in this approach i used like 8h token valid  / it was some inside project so this was fine. some renewing of them should be added if You want short timeout of token ;)

**Comment by W2426566602** [comment]
@d00lar @LoicVandersmissen @krunalshah10287 Hello everyone, I have also encountered such a demand now, it turns out that my project is pure blazor wasm + webapi and hosted on IIS, but recently the customer wants to give up the account password login and directly use AD information to log in without login operation; But because wasm runs on the browser side, the browser application is a sandbox, and you can't get AD information, but I think the author's project seems to be okay, how do I need to transform it;

**Comment by W2426566602** [comment]
@d00lar @LoicVandersmissen @krunalshah10287 I understand this code, it doesn't seem to solve my problem, because the web application running in the browser does not allow to get the system information (ad information) The author gets it through the API, but the API needs to be in the domain environment, but when you deploy the API application in IIS and access, a pop-up window will pop up to enter the domain account password; I can't find a way to do this for the time being @LoicVandersmissen That's the question you asked ；Because he can't get the domain information of the current link, he can only verify it through the pop-up window

**Comment by LoicVandersmissen** [comment]
Hello, my problem wasn't the obligation to logon with the popup window but a loop of opening on this popup even with a correct couple AD-login/Password even with the good authentication parameters on IIS.
I bypassed it by launching the application in kestrel mode instead of IIS.
